packages/editor/types.ts

```diff
@@ -37,6 +39,7 @@ export enum EditorCommandType {
 
 	// Editing and navigation commands
 	ScrollSelectionIntoView = 'scrollSelectionIntoView',
+	DeleteLine = 'deleteLine',
```

Comment:
For me, <kbd>ctrl</kbd>-<kbd>d</kbd> continues to select the current word, rather than deleting the current line. (I'm on Linux).

I suspect this is related to the default CodeMirror 6 keybindings (search for `...searchKeymap` in `createEditor.ts`).

Reply:
This does seem to be caused by `searchKeymap`. Removing `searchKeymap` fixes this issue, but it also breaks several search keybindings (e.g. `Esc` to close the search dialog).

Because it's a different issue, removing `searchKeymap` or fixing this issue in another way should probably be done in a different pull request.

Reply:
yeah will fix it in a separate pr then.
though Edit > Delete Line works.

Reply:
I've created an issue for it: https://github.com/laurent22/joplin/issues/10020